### PR TITLE
[Component with its own class name as service type] LPS-199148 Refactor BatchEngineImportStrategyFactory and ResourceBundleLoaderProvider component

### PR DESCRIPTION
**Pull request by Valmir-unicap**
- Ticket: https://liferay.atlassian.net/browse/LPS-199148
- Correction of: https://github.com/liferay-core-infra/liferay-portal/pull/5928
- Related PR: https://github.com/rafaprax/liferay-portal/pull/1336

### CODE REVIEW

**Comment by Valmir-unicap** [comment]
ci:test:relevant

**Comment by Valmir-unicap** [comment]
ci:test:startup

**Comment by catarineaguiar** [comment]
no regressions found

**Review by rafaprax — APPROVED** [review]

File: modules/apps/batch-engine/batch-engine-service/src/main/java/com/liferay/batch/engine/internal/BatchEngineImportTaskExecutorImpl.java [hunk 1]
```diff
@@ -165,6 +174,23 @@ private void _commitItems(
 			batchEngineImportTask);
 	}
 
+	private BatchEngineImportStrategy _create(
```
Comment (dantewang) [inline modules/apps/batch-engine/batch-engine-service/src/main/java/com/liferay/batch/engine/internal/BatchEngineImportTaskExecutorImpl.java:177]: @Valmir-unicap After merging, `_create` is too vague, please rename to `_createBatchEngineImportStrategy`.

**Comment by Valmir-unicap** [comment]
ci:test:startup

**Comment by zhangyan0701** [comment]
No regressions found.

**Review by dantewang — APPROVED** [review]

**Comment by dantewang** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/144622

cc @tinatian

**Pull request closed without merging**